controllers/articles.controllers.js

```diff
@@ -30,4 +30,18 @@ exports.getCommentsForArticle = (req,res,next) => {
         res.status(200).send({comments})
     })
     .catch(next)
+}
+
+exports.postCommentOnArticle = (req,res,next) => {
+
+    const { article_id } = req.params
+    const comment = req.body
+
+    fetchArticlesById(article_id).then((article) => {
```

Comment:
we don't need to fetch article by Id here because article_id is a foreign key in the comment table so if it doesn't exist an error will be thrown